sdk/netapp/arm-netapp/src/models/index.ts

```diff
@@ -1091,8 +1207,6 @@ export interface VolumeGroupMetaData {
   applicationIdentifier?: string;
   /** Application specific placement rules for the volume group */
   globalPlacementRules?: PlacementKeyValuePairs[];
-  /** Application specific identifier of deployment rules for the volume group */
-  deploymentSpecId?: string;
```

Comment:
Can you show me the link where this breaking has been approved in api layer ? Thanks

Reply:
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=3280454&view=logs&j=8f628bfe-7d8d-5bf3-e141-37106862c4f5&t=723dfc11-d28f-5d74-45f1-663cde6746b4&l=3471